src/_gettsim/parameters/erwerbsm_rente.yaml

```
@@ -0,0 +1,232 @@
+    "https://sozialversicherung-kompetent.de/rentenversicherung/leistungsrecht/
+    295-erwerbsminderungsrente-und-rentenabschlaege.html"
+    for 2014 reform see:
+    Artikel 1 Nr. 4 des RV-Leistungsverbesserungsgesetz vom 23. Juni 2014,
+    BGBl. I, Seite 787
+    for the time before 2004 see:
+    "https://www.bgbl.de/xaver/bgbl/text.xav?SID=&tf=xaver.component.
+    Text_0&tocf=&qmf=&hlf=xaver.component.Hitlist_0&bk=bgbl&start=%2F%2F*
+    %5B%40node_id%3D%27955873%27%5D&skin=pdf&tlevel=-2&nohist=1&sinst=23CF02AC"
+    (Anlage 23 (p.1837))
+  2001-01-01:
+    scalar: 62.916666
+  2001-02-01:
+    scalar: 62.833333
+  2001-03-01:
+    scalar: 62.75
+  2001-04-01:
+    scalar: 62.666666
+  2001-05-01:
+    scalar: 62.583333
+  2001-06-01:
+    scalar: 62.5
+  2001-07-01:
+    scalar: 62.416666
+  2001-08-01:
+    scalar: 62.333333
+  2001-09-01:
+    scalar: 62.25
+  2001-10-01:
+    scalar: 62.166666
+  2001-11-01:
+    scalar: 62.083333
+  2001-12-01:
+    scalar: 62
+  2002-01-01:
+    scalar: 61.916666
+  2002-02-01:
+    scalar: 61.833333
+  2002-03-01:
+    scalar: 61.75
+  2002-04-01:
+    scalar: 61.666666
+  2002-05-01:
+    scalar: 61.583333
+  2002-06-01:
+    scalar: 61.5
+  2002-07-01:
+    scalar: 61.416666
+  2002-08-01:
+    scalar: 61.333333
+  2002-09-01:
+    scalar: 61.25
+  2002-10-01:
+    scalar: 61.166666
+  2002-11-01:
+    scalar: 61.083333
+  2002-12-01:
+    scalar: 61
+  2003-01-01:
+    scalar: 60.916666
+  2003-02-01:
+    scalar: 60.833333
+  2003-03-01:
+    scalar: 60.75
+  2003-04-01:
+    scalar: 60.666666
+  2003-05-01:
+    scalar: 60.583333
+  2003-06-01:
+    scalar: 60.5
+  2003-07-01:
+    scalar: 60.416666
+  2003-08-01:
+    scalar: 60.333333
+  2003-09-01:
+    scalar: 60.25
+  2003-10-01:
+    scalar: 60.166666
+  2003-11-01:
+    scalar: 60.083333
+  2003-12-01:
+    scalar: 60
+  2014-07-01:
+    scalar: 62
+  2019-01-01:
+    scalar: 65.666666
+  2020-01-01:
+    scalar: 65.75
+  2021-01-01:
+    scalar: 65.833333
+  2022-01-01:
+    scalar: 65.916666
+  2023-01-01:
+    scalar: 66
+  2024-01-01:
+    scalar: 66.083333
+  2025-01-01:
+    scalar: 66.166666
+  2026-01-01:
+    scalar: 66.25
+  2027-01-01:
+    scalar: 66.333333
+  2028-01-01:
+    scalar: 66.5
+  2029-01-01:
+    scalar: 66.666666
+  2030-01-01:
+    scalar: 66.833333
+  2031-01-01:
+    scalar: 67
+min_zugangsfaktor_erwerbsm_rente:
+  name:
+    de: Kleinster möglicher Zugangsfaktor bei der Erwerbsminderungsrente
+    en: Smallest possible access factor for public disability insurance
+  description:
+    de: >-
+      Abzüge wegen frühzeitigem Eintritt in die Erwerbsminderungsrente
+      sind bei 10.8% gedeckelt.
+    en: Deductions for early retirement are capped at 10.8%.
+  unit: Factor
+  reference: § 77 SGB VI Zugangsfaktor
+  note: >-
+    Rentenreformgesetz 1999 (RRG 1999)
+    "http://www.portal-sozialpolitik.de/index.php?page=rentenversicherung"
+  2000-01-01:
+    scalar: 0.892
+rentenartfaktor_erwerbsm_rente:
```

```suggestion
rentenartfaktor:
```
The params object is already called `erwerbsm_rente`, no need to double up.